lib/messaging/messagehandler.rb

```diff
@@ -0,0 +1,151 @@
+require 'json'
+require 'time'
+require './lib/database/dbaccess.rb'
+
+require_relative 'messagehandler_emergencystop.rb'
+require_relative 'messagehandler_logs.rb'
+require_relative 'messagehandler_measurements.rb'
+require_relative 'messagehandler_message.rb'
+require_relative 'messagehandler_parameters.rb'
+require_relative 'messagehandler_schedule.rb'
+require_relative 'messagehandler_status.rb'
+
+
+# Get the JSON command, received through skynet, and send it to the farmbot
+# command queue Parses JSON messages received through SkyNet.
+class MessageHandler
+
+  attr_accessor :message
+  attr_accessor :use_test_handler
+
+  ## general handling messages
+
+  def initialize
+    #@dbaccess = DbAccess.new
+    @dbaccess = $dbaccess
```

Comment:
Started from older version and afraid missed that one in the merge.
On 23 Mar 2015 13:39, "Rick Carlino" <notifications@github.com> wrote:

> In lib/messaging/messagehandler.rb
> <https://github.com/FarmBot/farmbot-raspberry-pi-controller/pull/77#discussion_r26932527>
> :
>
> > +require_relative 'messagehandler_schedule.rb'
> > +require_relative 'messagehandler_status.rb'
> > +
> > +
> > +# Get the JSON command, received through skynet, and send it to the farmbot
> > +# command queue Parses JSON messages received through SkyNet.
> > +class MessageHandler
> > +
> > +  attr_accessor :message
> > +  attr_accessor :use_test_handler
> > +
> > +  ## general handling messages
> > +
> > +  def initialize
> > +    #@dbaccess = DbAccess.new
> > +    @dbaccess = $dbaccess
>
> Was this a copy/paste job from an older commit? Looks like the $dbaccess
> global variable is back after being removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FarmBot/farmbot-raspberry-pi-controller/pull/77/files#r26932527>
> .
>